ItemLevelDownloader/Engine/logger.ts

```diff
@@ -4,42 +4,73 @@ import { ArtifactItemStore } from '../Store/artifactItemStore';
 
 export class Logger {
 
-    constructor(verbose: boolean) {
-        this.verbose = verbose
+    constructor(store: ArtifactItemStore, verbose: boolean) {
```

Comment:
do we need to pass verbose to constructor?

Reply:
removed.